compiler/rustc_trait_selection/src/traits/select/confirmation.rs

```diff
@@ -48,12 +48,12 @@ impl<'cx, 'tcx> SelectionContext<'cx, 'tcx> {
         let mut impl_src = match candidate {
             BuiltinCandidate { has_nested } => {
```

Comment:
for future PR, we should do the same for old style candidates :grin: 

have `BuiltinCandidate` with a `CandidateKind`. Definitely not in this PR